platformcomponents/desktop/attachment-file.json

```diff
@@ -0,0 +1,28 @@
+{
+  "attachment": {
+    "file": {
+      "comment": "Applied to file attachments",
+      "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=5415%3A0",
+      "border": {
```

Comment:
we should not add "background" in all the elements, especially not in a border element